sos/report/plugins/libvirt.py

```diff
@@ -89,4 +89,25 @@ def postproc(self):
         for path_exp in libvirt_path_exps:
             self.do_path_regex_sub(path_exp, match_exp, r"\1******\3")
 
+
+class RedHatLibvirt(Libvirt, RedHatPlugin):
+
+    def setup(self):
+        super(RedHatLibvirt, self).setup()
+        self.add_copy_spec([
+            "/var/lib/openstack/config/libvirt",
+            "/var/lib/openstack/containers/libvirt*.json",
+        ])
+
+        if not self.get_option("all_logs"):
+            self.add_copy_spec([
+                "/var/log/containers/qemu/*.log",
+                "/var/log/containers/libvirt/*.log",
+            ])
+        else:
+            self.add_copy_spec([
+                "/var/log/containers/qemu/",
+                "/var/log/containers/libvirt/",
+            ])
```

Comment:
I don't think we need to split this out.

The `/var/lib/openstack` paths should, arguably, be collected by one of the openstack plugins. However, assuming we keep the collection in the `libvirt` plugin, we can simply add these to the existing collection lists. If they don't exist on a particular system, we skip them gracefully.

Further, the `/var/log/containers` paths fit nicely in with what is already called in the existing `IndependentPlugin` structure.

Reply:
I thought it was redundant to create an `openstack_libvirt` plugin since i would basically be copying most of this one, but i agree that we can avoid to split the logic. Let me fix it.